OutOfSchool/OutOfSchool.DataAccess/Repository/IProviderRepository.cs

```diff
@@ -1,6 +1,8 @@
 using OutOfSchool.Services.Models;
 using System.Threading.Tasks;
 using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
```

Comment:
(Code style) Excessive using. My Visual Studio shows it as an error and doesn't compile. 
Please share what IDE or way of build are you using that you don't get this error.

Same at least in `ProviderRepository`

Reply:
> (Code style) Excessive using. My Visual Studio shows it as an error and doesn't compile. Please share what IDE or way of build are you using that you don't get this error.
> 
> Same at least in `ProviderRepository`

Fixed excessive 'using' directives. Initially, I did not notice any red highlights in Visual Studio 2022, but the project still compiled successfully.